examples/resnet.py

```diff
@@ -98,13 +98,13 @@ def main(dataset, architecture, inference, train_batch_size, test_batch_size, lo
         prior_kwargs["expose_modules"] = [net.fc]
         if inference == "last-layer-mean-field":
             guide = functools.partial(tyxe.guides.AutoNormal,
-                                      init_loc_fn=tyxe.guides.PretrainedInitializer.from_net(net), init_scale=1e-4)
+                                      init_loc_fn=tyxe.guides.PretrainedInitializer.from_net(net, prefix="net"), init_scale=1e-4)
```

Comment:
I have not been able to test this, as I'm not sure how the pretrained weights are to be passed to the model.